frappe/__init__.py

```diff
@@ -89,6 +89,27 @@
 	warnings.simplefilter("always", PendingDeprecationWarning)
 
 
+def _get_local_proxy(self: Local, name: str) -> LocalProxy:
+	"""Get local proxy object by name."""
+
+	_local_contextvar = self._Local__storage
+
+	def _get_current_object() -> Any:
+		obj = _local_contextvar.get(None)
+
+		if obj is not None and name in obj:
+			return obj[name]
+
+		raise RuntimeError("object is not bound") from None
+
+	lp = LocalProxy(_local_contextvar, name)
```

Comment:
```suggestion
	lp = LocalProxy(_local_contextvar, name)
```

This is technically wrong. ContextVar can only hold one value and `LocalProxy` wants to manage it's own storage. 

It seems to be working only because `_get_current_object` is also overridden.

Reply:
Ig there's no way to avoid this without adding extra costs.